starky/src/stark_gen.rs

```diff
@@ -174,13 +175,13 @@ impl<F: FieldExtension> StarkContext<F> {
     }
 }
 
-#[derive(Default, Debug, PartialEq)]
+#[derive(Default, Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
We should override the default serialization for StarkProof.  Just check out the serialization in serializer.rs